README.md

```diff
@@ -125,19 +152,31 @@ And let's get going! 🚀
    git log --pretty=fuller
    ```
 
+   A **commit** is an individual collection of changes to the repository.
```

Comment:
Add that it represents a given state of the repo. I compare a commit to a save file or checkpoint in a video game.